lib/drag_time_api_web/router.ex

```diff
@@ -11,6 +11,7 @@ defmodule DragTimeApiWeb.Router do
     resources "/locations", LocationController, except: [:new, :edit]
     resources "/events", EventController, except: [:new, :edit]
     resources "/artists", ArtistController, except: [:new, :edit]
+    resources "/access", AccessController, execpt: [:new, :edit]
```

Comment:
You could probably do only: [:index] since that's the only we are using right now.

Reply:
That is a good point, I don't think we will ever need the edit route